index.html

```diff
@@ -36,5 +36,147 @@ <h5>LET'S CONNECT</h5>
+            <p class="intro-text">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="socials">
+                <h5>LET'S CONNECT</h5>
+                <ul class="social-icon">
+                    <li> <a href="#"> <img src="./assets/twitter.svg" alt="twitter"> </a> </li>
+                    <li> <a href="#"> <img src="./assets/Linkedin icon.svg" alt="Linkedin"> </a> </li>
+                    <li> <a href="#"> <img src="./assets/medium.svg" alt="medium"> </a> </li>
+                    <li> <a href="#"> <img src="./assets/github.svg" alt="github"> </a> </li>
+                    <li> <a href="#"> <img src="./assets/angellist.svg" alt="angellist"> </a> </li>
+                </ul>
+            </div>
+            <button type="button" class="btn-detail resume">Get my resume</button>
+        </main>
+        
+        <main class="skills">
+            <div class="dropdown">
+                <div class="lang">
+                    <h3>Languages</h3>
+                    <img src="./assets/arrow.png" alt="arrow">
+                </div>
+                <div class="list">
+                    <div class="item">
+                        <img src="./assets/lang1.png" alt="lang1">
+                        <p>JavaScript</p>
+                    </div>
+                    <div class="item">
+                        <img src="./assets/lang2.png" alt="lang2">
+                        <p>HTML</p>
+                    </div>
+                    <div class="item">
+                        <img src="./assets/lang3.png" alt="lang3">
+                        <p>CSS</p>
+                    </div>
+                </div>
+            </div>
+            <!-------- Frameworks ----->
+            <div class="dropdown border">
+                <div class="lang">
+                    <h3>Frameworks</h3>
+                    <img src="./assets/arrow2.png" alt="arrow">
+                </div>
+            </div>
+            <!-------- Frameworks ----->
+            <div class="dropdown border">
+                <div class="lang">
+                    <h3>Skills</h3>
+                    <img src="./assets/arrow2.png" alt="arrow">
+                </div>
+            </div>
+        </main>
+    </section>
+    <footer class="contact-me">
+        <main class="shapes">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+            <input type="text" class="input-col">
+            <input type="email" class="input-col">
+            <textarea class="input-col comment" placeholder="Write your message here..."></textarea>
+            <button type="button" class="btn-detail contact">Get in touch</button>
```

Comment:
- [ ] I can see that you did a great job on this project, but they're still to improve on something for your project to look more professional, I can see that you forget to put a `placeholder` in your form to refer a user what to complete in the form, kindly add it for a good user experience,  kindly fix this :point_down: don't forget to follow figma guidelines, make sure you put the same `font` as for Figma

Your design
![Screenshot from 2023-04-28 00-03-06](https://user-images.githubusercontent.com/88894525/235001073-4df6cc69-2f2e-432d-9c4d-e0c5e3eb6c4c.png)

Figma Design
![Screenshot from 2023-04-27 23-56-38](https://user-images.githubusercontent.com/88894525/235001147-4fdde430-42d4-4d2d-b9db-1276eee4c8e3.png)